cmd/githubrunnerscalesetlistener/main.go

```diff
@@ -71,23 +61,28 @@ func main() {
 		os.Exit(1)
 	}
 
+	if len(c.RunnerRepository) == 0 || len(c.RunnerOrg) == 0 || len(c.Token) == 0 || len(c.RunnerScaleSetName) == 0 {
+		fmt.Fprintln(os.Stderr, "GitHub config is not provided:", c.RunnerRepository, c.RunnerOrg, c.Token, c.RunnerScaleSetName)
```

Comment:
I agree we should consolidate logging approaches. I will prefer to defer this to the "productionize" phase of development though. At the very least, there are enough places to change logging approach that it will be better in a different PR.